compiler/luci/pass/src/ResolveCustomOpAddPass.test.cpp

```diff
@@ -34,3 +170,61 @@ TEST_F(ResolveCustomOpAddPassTest, name)
   auto const name = _pass.name();
   ASSERT_NE(nullptr, name);
 }
+
+TEST_F(ResolveCustomOpAddPassTest, simple_test_CustomBroadcastTo)
+{
+  _g.init(true);
+
+  auto ret = _pass.run(_g.g());
+  EXPECT_EQ(true, ret);
+
+  auto add = dynamic_cast<luci::CircleAdd *>(_g.output(0)->from());
+  EXPECT_NE(nullptr, add);
+
+  // if this pass runs, x and y of Custom(AddV2) will be connected to x and y of Add.
+  // so, check if x and y of Custom(AddV2) Add are the equal to x and y of Add.
+  EXPECT_EQ(_g.input_x(), add->x());
```

Comment:
Is this must have? or good to have?

Reply:
@seanshpark 

> Is this must have? or good to have?

I think this is good to have.
This is because i think we can check the following functions in the pass.

1/ x of Add is set to CircleInput. 
2/ y of Add is set to BroadcastTo->input().  


```
=========ResolveCustomOpAddPass.cpp==========
...
...
  add->x(addv2->inputs(1 - broadcastTo_idx));
  add->y(bc_input);
```

Reply:
Well noted. Thank you :)